api/utils/requestProcessor.js

```diff
@@ -1424,8 +1424,26 @@ const processRequest = (params) => {
                 case 'permissions':
                     validateRead(params, 'core', function() {
                         var features = ["core", "events" /* , "global_configurations", "global_applications", "global_users", "global_jobs", "global_upload" */];
-                        plugins.dispatch("/permissions/features", {params: params, features: features}, function() {
-                            common.returnOutput(params, features);
+                        /*
+                            Example structure for featuresPermissionDependency Object
```

Comment:
Can you add other keys too
not just 'c'.
example should be more comprehensive that generally covers as many cases as possible for an implementation